src/libmongoc/examples/client-side-encryption-auto-decryption.c

```diff
@@ -142,7 +142,7 @@ main (void)
    to_encrypt.value_type = BSON_TYPE_UTF8;
    to_encrypt.value.v_utf8.str = "123456789";
    const size_t len = strlen (to_encrypt.value.v_utf8.str);
-   BSON_ASSERT (bson_in_range_unsigned (uint32_t, len));
+   BSON_ASSERT (len >= 0 && (uint64_t) len <= UINT32_MAX);
```

Comment:
```suggestion
   BSON_ASSERT (len <= UINT32_MAX);
```
```
warning: -Wtype-limits - comparison of unsigned expression in ‘>= 0’ is always true
```
Consistent signedness -> cast not required.